packages/ui/src/components/editor/properties/text/index.tsx

```diff
@@ -192,8 +194,7 @@ export const TextNodeEditor: EditorComponentType = (props) => {
             <SelectInput
               options={SelectOptions.TextAlignment}
               value={text.textAlign.value}
-              onChange={commitProperty(TextComponent, 'textAlign')}
-              //onRelease={commitProperty(TextComponent, 'textAlign')}
+              onChange={(value) => updateProperty(TextComponent, 'textAlign')(value.toString() as TroikaTextAlignment)}
```

Comment:
without commit property we will only see this value updated real time in simulation. Commit property commits changes into the authoring layer of the editor and is needed to save changes.

Reply:
good to know, thanks - I originally changed from commitProperty to updateProperty since I wasn't seeing the changes reflected in the studio simulation

I've now changed to both call both updateProperty and commitProperty in these cases